docs/src/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+title: Cash App Pay
+parent: Getting Started
+nav_order: 4
+---
+
+# Cash App Pay
+{: .d-inline-block .no_toc }
+NEW (v5.3.0)
+{: .label .label-green }
+
+
+<details markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+{: .alert }
+> Cash App Pay is currently available in the following region(s): US
+
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK.
```

Comment:
```suggestion
You can add Cash App Pay support using your Afterpay merchant account. To do this, you must generate a token by sending a server-to-server call to the Afterpay API Create Checkout endpoint<!---pls add the link here--> with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK. <!--same comment as for the android docs, can we change the first line to specify the support?--->
```

Reply:
addressed in [801a425](https://github.com/afterpay/sdk-ios/pull/244/commits/801a425bf07098f6bdb4618fde0b6b967e0f00a0)

Reply:
Please change the first line to "With our latest enhancements, you can now support taking Cash App Pay payments using your Afterpay merchant account."

Reply:
@ScottAntonacAP in the last line "PayKit" must have a space between the words. Pay Kit